test/razorpay/test_generic.rb

```diff
@@ -0,0 +1,112 @@
+require 'test_helper'
+
+module Razorpay
+  # Tests for Razorpay::Addon
+  class RazorpayGenericTest < Minitest::Test
+    class Generic < Razorpay::Entity; end
+
+    def setup
+      @order_id = 'order_50sX9hGHZJvjjI'
+      @customer_id = 'cust_6vRXClWqnLhV14'
+      @invoice_id = 'inv_6vRZmJYFAG1mNq'
+      @addon_id = 'ao_IrSY3UIqDRx7df'
+    end
+
+    def test_generic_should_be_defined
+      refute_nil Razorpay::Generic
+    end
+   
+    # Test fetch endpoint by order entity
+    def test_geneic_orders_should_be_fetch    
+      stub_get(%r{orders/#{@order_id}$}, 'fake_order')
+
+      client = Razorpay::Generic.new("orders")
+      
+      order = client.do(@order_id)
+
+      assert_equal 5000, order.amount
+      assert_equal 'INR', order.currency
+      assert_equal 'TEST', order.receipt
+    end
+
+    # Test post endpoint by order entity
+    def test_geneic_orders_should_be_create
```

Comment:
Please correct the spelling `test_generic_orders_should_be_create` in all the test case names.